src/app/user-page/user-page.component.html

```diff
@@ -121,6 +121,49 @@ <h4 matLine class="accent-1-dark mt-2 mb-0">{{ user?.username }}</h4>
         <app-recent-events></app-recent-events>
       </div>
     </mat-tab>
+    <mat-tab *ngIf="loggedInUserIsAdminOrCurator">
+      <ng-template mat-tab-label>
+        <div fxLayoutGap="0.5rem">
+          <span>Other Linked Accounts</span>
+        </div>
+      </ng-template>
+      <div class="p-2">
+        <div *ngFor="let token of tokens">
+          <mat-card *ngIf="token.tokenSource === 'quay.io'" data-cy="other-linked-accounts-quay">
+            <mat-card-header>
+              <div mat-card-avatar matBadge="✓" class="mr-4">
+                <img src="../../assets/images/account-logos/quay.svg" alt="Quay.io logo" class="w-100" />
+              </div>
+              <mat-card-title class="normal-line-height">Quay.io Account</mat-card-title>
```

Comment:
Not sure if it's a privacy issue, but I think it would also be nice to see the username associated with the account

Reply:
@denis-yuen any thoughts?

Reply:
Since this is admin and curator only, I think it would be fine. 
The actual endpoint would be protected anyway

Reply:
added